File: kattis/skocimis.py
# https://open.kattis.com/problems/skocimis
# 1s
# 1.6 Easy

# 52 chars: It would have placed it as place 8 in Top 10, my Prolog program (52 chars) is there already
a,b,c=map(int,input().split())
print(max(b-a,c-b)-1)

File: kattis/skocimis.rb
# https://open.kattis.com/problems/skocimis
# 1s
# 1.6 Easy

# 46 chars: This would have placed me in Top 10 place 2, but I want my Prolog program (53 chars place 8) to be in the list..
a,b,c=gets.split.map(&:to_i)
p [b-a,c-b].max-1


